packages/backend-app-api/src/config/config.ts

```diff
@@ -109,11 +111,15 @@ export async function loadBackendConfig(options: {
       }),
     },
   });
-
   console.info(
     `Loaded config from ${appConfigs.map(c => c.context).join(', ')}`,
   );
 
+  // add the additional config if provided
+  if (options.additionalConfig) {
+    appConfigs.push(options.additionalConfig);
```

Comment:
This needs to be merged into the `newConfigs` in the `onChange` callback above too.

Reply:
Good catch